core/cmd/shell.go

```diff
@@ -56,6 +57,12 @@ import (
 	"github.com/smartcontractkit/chainlink/v2/plugins"
 )
 
+func init() {
+	// hack to undo geth's disruption of the std default logger
+	// remove with geth v1.13.10
+	slog.SetDefault(slog.New(slog.NewTextHandler(os.Stderr, nil)))
```

Comment:
Geth is registering a discard logger from an `init` that we have to replace to restore standard logs for goose, etc.: https://github.com/jwasinger/go-ethereum/blob/38926cc6e507ba622275de789800e74215117dec/log/root.go#L13-L14